snovault/schema_utils.py

```diff
@@ -82,13 +82,13 @@ def favor_app_specific_schema_ref(schema_ref: str) -> str:
     def json_file_contains_element(json_filename: str, json_element: str) -> bool:
         """
         If the given JSON file exists and contains the given JSON element name then
-        returns True, otherwise returnes False. The given JSON element may or may 
+        returns True, otherwise returnes False. The given JSON element may or may
```

Comment:
Semicolon before "otherwise" and comma after. I think recent versions of PyCharm probably remind you of things like this, as they seem to have connected a grammar checker.

```suggestion
        returns True; otherwise, returns False. The given JSON element may or may
```

Fixes typo "returnes", too.